tt_eager/tt_dnn/op_library/bmm/bmm_op.cpp

```diff
@@ -331,15 +331,16 @@ tt::operations::primary::MatmulMultiCoreReuseMultiCast1DProgramConfig get_mcast_
     auto out_subblock_w = std::get<1>(subblock_hw);
 
     return tt::operations::primary::MatmulMultiCoreReuseMultiCast1DProgramConfig{
-        .compute_with_storage_grid_size = grid_size,
-        .in0_block_w = in0_block_w,
-        .out_subblock_h = out_subblock_h,
-        .out_subblock_w = out_subblock_w,
-        .per_core_M = per_core_M,
-        .per_core_N = per_core_N,
-        .fuse_batch = fuse_batch,
-        .fused_activation = fused_activation,
-        .mcast_in0 = mcast_in0};
+        grid_size,
```

Comment:
Is there a reason why we are removing named struct memebers?

Reply:
Cause it's only allowed if we use default constructor